src/generic/video-modal/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { ModalDialog } from '@openedx/paragon';
+
+import { VideoModalProps } from './types';
+import messages from './messages';
+
+export const VideoModal = ({ isOpen, close, videoID }: VideoModalProps) => {
+  const intl = useIntl();
+
+  return (
+    <ModalDialog
+      title={intl.formatMessage(messages.videoModalTitle)}
+      size="lg"
```

Comment:
Let's change this hardcoded value to `size={size}` and specify a default value for `size` prop



```suggestion
      size={size}
```

Reply:
fixed